_data/gaming.yml

```diff
@@ -2,298 +2,269 @@ websites:
+      img: leagueoflegends.png
+      bcc: No
       twitter: leagueoflegends
       facebook: leagueoflegends
-      img: leagueoflegends.png
-      tfa: No
 
     - name: Mojang
       url: https://mojang.com/
-      twitter: MojangSupport
       img: mojang.png
-      tfa: No
+      bcc: No
+      twitter: MojangSupport
 
     - name: Nintendo Network
       url: https://www.nintendo.com/
+      img: nintendo.png
+      bcc: No
       twitter: NintendoAmerica
       facebook: Nintendo
-      img: nintendo.png
-      tfa: No
 
     - name: Origin
       url: https://www.origin.com/
       img: origin.png
-      tfa: Yes
-      email: Yes
-      sms: Yes
-      software: Yes
-      doc: https://help.ea.com/en-us/help/account/origin-login-verification-information/
+      bcc: No
+      twitter: OriginInsider
+      facebook: OriginInsider
 
     - name: Playstation Network
       url: https://www.playstation.com/
       img: psn.png
-      tfa: Yes
-      sms: Yes
-      doc: https://www.playstation.com/en-us/account-security/2-step-verification/
+      bcc: No
+      twitter: PlayStation
+      facebook: PlayStation
 
     - name: Roberts Space Industries
       url: https://robertsspaceindustries.com/
       img: robertsspaceindustries.png
-      tfa: Yes
-      software: Yes
-      email: Yes
-      doc: https://robertsspaceindustries.com/faq/two-step-authentication
+      bcc: No
+      twitter: RobertsSpaceInd
+      facebook: RobertsSpaceIndustries
 
     - name: Rockstar Games Social Club
       url: https://socialclub.rockstargames.com
+      img: rockstargames.png
+      bcc: No
       twitter: RockstarSupport
       facebook: rockstargames
-      img: rockstargames.png
-      tfa: No
 
     - name: RuneScape
       url: http://www.runescape.com/
       img: runescape.png
-      tfa: Yes
-      software: Yes
-      doc: https://support.runescape.com/hc/en-gb/articles/206811835-Setting-the-Authenticator-Mobile-Device
+      bcc: No
+      twitter: RuneScape
+      facebook: RuneScape
 
     - name: Square Enix
       url: http://na.square-enix.com/
       img: squarenix.png
-      tfa: Yes
-      software: Yes
-      hardware: Yes
-      doc: http://www.square-enix.com/na/account/otp/
-      exceptions:
-          text: "Software implementation requires you to have purchased either Final Fantasy XI or Final Fantasy XIV and registered the game."
+      bcc: No
+      twitter: SquareEnix
+      facebook: SquareEnix
 
     - name: "Star Wars: The Old Republic"
       url: http://www.swtor.com/
       img: swtor.png
-      tfa: Yes
-      software: Yes
-      hardware: Yes
-      doc: http://www.swtor.com/info/security-key
+      bcc: No
+      twitter: swtor
+      facebook: starwarstheoldrepublic
 
     - name: Stardock Corporation
       url: https://www.stardock.com/
+      img: stardock.png
+      bcc: No
       twitter: Stardock
       facebook: official.stardock
-      img: stardock.png
-      tfa: No
 
     - name: Steam
```

Comment:
Steam accepts BTC, I think it should have that tagged. Please fix and then all looks good

Reply:
thanks, fixed.